src/main/java/uk/gov/crowncommercial/dts/scale/cat/model/jaggaer/RfxTemplateMapping.java

```diff
@@ -13,7 +13,7 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @Builder
-@FieldDefaults(level = AccessLevel.PRIVATE)
+@FieldDefaults(level = AccessLevel.PUBLIC)
```

Comment:
I can't see why you need to make the change from private to public. Is it because you are accessing fields directly? Or for some other reason?